format/hint/schema.json

```diff
@@ -0,0 +1,19 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "type": "object",
+  "properties": {
+    "id": {
+      "type": "string"
+    },
+    "value": {
```

Comment:
Me too.
But the Exercise uses `value` and it's simplier to change the schema for now.
Also we will need to change the Hint schema later to allow any type of content instead of only text.